### PR TITLE
Add Scene platform and category to  current Overkiz docs

### PR DESCRIPTION
**Pull request by iMicknl**
## Proposed change
Adds scene platform and category to  current Overkiz docs

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/62884
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by bdraco — APPROVED** [review]
Thanks 👍

File: source/_integrations/overkiz.markdown [hunk 1]
```diff
@@ -7,6 +7,7 @@ ha_category:
   - Button
   - Lock
   - Number
+  - Scene
 ha_release: 2022.1.0
```
Comment (bdraco) [inline source/_integrations/overkiz.markdown:11]:
```suggestion
ha_release: 2022.2.0
```

Comment (bdraco) [reply source/_integrations/overkiz.markdown:11]: For a separate PR

Edit: Looks like also commented here https://github.com/home-assistant/home-assistant.io/pull/20841#discussion_r775367597

Comment (iMicknl) [reply source/_integrations/overkiz.markdown:11]: https://github.com/home-assistant/home-assistant.io/pull/20881

**Pull request merged**